app/views/citationInfo.html

```diff
@@ -3,6 +3,7 @@
 </div>
 <div class="sub-page-container">
   <div class="ticket-results-container container-fluid topMargin alignContainerToPage">
+    <div class="well well-sm warning-red-text" ng-if="ctrl.issueMultiplePeopleWarning()">Multiple people may have been returned for the tickets.  Please make sure you check that you are looking at the correct ticket.</div>
```

Comment:
This wording is a little less than ideal (doesn't read clearly). Maybe "The results may contain tickets for multiple people. Please make sure you look closely for the correct ticket"?